.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up MySQL
+      uses: mirromutth/mysql-action@v1.1
+      with:
+        mysql database: 'sharkey_test'
+        mysql root password: 'root'
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17
```

Comment:
nice bump to latest Go! 🎉  (I believe we were still running on Go 1.13 in Travis CI 😭)